frontend/lib/norent/letter-builder/letter-preview.tsx

```diff
@@ -52,35 +52,52 @@ const SendLetterModal: React.FC<{
 export const NorentLetterPreviewPage = MiddleProgressStep((props) => {
   const { letterContent } = NorentRoutes.locale;
   const { session } = useContext(AppContext);
-  const showLetterPreview =
-    session.norentScaffolding?.hasLandlordMailingAddress;
-  const showEmailPreview = session.norentScaffolding?.hasLandlordEmailAddress;
+  const isMailingLetter = session.norentScaffolding?.hasLandlordMailingAddress;
+  const isEmailingLetter = session.norentScaffolding?.hasLandlordEmailAddress;
```

Comment:
Actually instead of pulling from these can you test the truthiness of the actual model data at `session.landlordDetails.email` and `session.landlordDetails.address`?  The `norentScaffolding` variables you're using are really just meant to remember whether to ask the user for the information but the real "source of truth" that the backend works off is in `landlordDetails`.

Reply:
Cool cool, will do!